ovirt/resource_ovirt_vm.go

```diff
@@ -174,6 +174,12 @@ func resourceOvirtVM() *schema.Resource {
 							Required: true,
 							ForceNew: true,
 						},
+						"storage_domain": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: false,
+							Default:  "",
```

Comment:
Why set a default which is invalid?

Reply:
Default value "" - in this case this is just indicative variable with string type.
As any disk always have storage domain parameter I've specified empty string as default value.
This does not contradict the logic implemented in the code